### DISCUSSION TITLE
How to make shared remote chest?

### DISCUSSION THREAD

**Discussion by FoliageOwO** [stefvanschie/IF#254] [Q&A]
I'm trying to make a plugin which can allow player store their items into a remote chest, and I wanna make it shared with other players, just like a `public chest`.
Is there a way using `InventoryFramework` to implement it?

**Comment by stefvanschie** [comment]
By default if you open the gui for multiple people, the gui will be synchronized between these people, so they'll all see the same thing. As for storing the items they put in, that's something you'll need to do yourself.
